dashboard/src/components/Header/Header.tsx

```diff
@@ -83,6 +88,9 @@ class Header extends React.Component<IHeaderProps, IHeaderState> {
       this.state.configOpen ? "header__nav__submenu-open" : ""
     }`;
 
+    const reposPath = this.props.featureFlags.reposPerNamespace
+      ? `/config/ns/${namespace.current}/repos`
+      : "/config/repos";
```

Comment:
isn't it a bit easier if we just change the route to `/config/ns/{namespace}/repos` and ignore the namespace in the route in the RepoList component if the flag is not enabled?

Reply:
Yes, this is exactly what I did originally, but based on your review of 1498 (in particular, [this thread](https://github.com/kubeapps/kubeapps/pull/1498#pullrequestreview-352830996)) from which I assumed you were keen to ensure that we don't change current behaviour even if it's not visible to the use, I updated to ensure this change was behind the feature flag also. Maybe I read to much into the comment?

Reply:
Yes, my comment there was because that was an usability issue (some without permissions to create ClusterRoles wouldn't be able to install the chart). In this case there is no usability issue, just a change in the route. In any case, I am fine with this change.

Reply:
Ah - I wouldn't have thought it likely that someone had perms to create CRDs but not a ClusterRole, but possible, sure. Anyway, worked out well, it was just a bit bigger than expected. Thanks!